code-challenges.test.js

```diff
@@ -16,14 +16,51 @@
 
 const fibonacciLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
-
 const fibonacciLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+describe("figFunct", () => {
+  it("takes in a number (greater than 2)", () => {
+    expect(figFunct(6)).toEqual([1, 1, 2, 3, 5, 8]);
+  })
+})
+describe("figFunct", () => {
+  it("takes in a number (greater than 2)", () => {
+    expect(figFunct(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55]);
+  })
+})
+
+//----------> figFunct is not defined
+
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
+// f(0)= 0 applies only to the first integer
+// f(1)= 1 applies only to the second integer
+// f(n)= f(n-1) + f(n-2)
+//Create a function figFunct 
```

Comment:
Definitely need to elaborate more on this pseudo code. Make sure to explain the code step by step